pintc/src/main.rs

```diff
@@ -103,6 +103,49 @@ fn main() -> anyhow::Result<()> {
                 }
             };
 
+            // Compute and print the content addresses for the contract and its predicates
+            let compiled_predicate_contract_addresses: Vec<_> = compiled_contract
+                .predicates
+                .iter()
+                .map(essential_hash::content_addr)
+                .collect();
+            let ca = essential_hash::contract_addr::from_predicate_addrs(
+                compiled_predicate_contract_addresses.iter().cloned(),
+                &args.salt.unwrap_or_default(),
+            );
+
+            let pinned_name = filepath
+                .file_stem()
+                .expect("Failed to get file stem")
+                .to_str()
+                .expect("Invalid unicode in contract name");
+            let name_col_w = {
+                let mut name_w = 0;
+                for name in &compiled_contract.names {
+                    let w = name.chars().count();
+                    name_w = std::cmp::max(name_w, w);
+                }
+                pinned_name.chars().count() + name_w
+            };
+            let bold = Style::new().bold();
+            println!(
+                "    {}contract{} {:<name_col_w$} {}",
```

Comment:
This will always be `contract`, correct?

Reply:
Yeah pretty much.. for now.